index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <style>
+      @media screen and (max-width: 700px){
+        p{
+          font-size: 20px;
+        }
+        body
+        {
+            display: flex;
+        }
+      }
+    </style>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+      <header class="header" id="Documentation">Documentation</header>
+      <ul>
+      <li id="nav-list"><a href="#introduction" class="nav-link">Introduction</a></li>
+      <li id="nav-list"><a href="#what_should_we_already_know" class="nav-link">what should we already know</a></li>
+      <li id="nav-list"><a href="#javascript_and_java" class="nav-link">Javascript and java</a></li>
+      <li id="nav-list"><a href="#hello_world" class="nav-link">Hello world</a></li>
+      <li id="nav-list"><a href="#variables" class="nav-link">variables</a></li>
+      <li id="nav-list"><a href="#declaring_variables" class="nav-link">Declaring variables</a></li>    
+      </ul>
+    </nav>
+
+    <main id="main-doc">      
+      <section id="introduction" class="main-section">
+        <header>
+          Introduction
+        </header>
+
+        <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+        <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+          <ul>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+          </ul>
+       
+      </section>
+
+      <section id="what_should_we_already_know" class="main-section">
+        <header>
+          What should we already know
+          
+        </header>
+        <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+        <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+          <ul>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+          </ul>
+      </section>
+
+      <section id="javascript_and_java" class="main-section">
+        
+        <header>
+          Javascript and Java
+          
+        </header>
+        <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+        <p><code>Lorem ipsum dolor sit, amet consectetur adipisici.</code></p>
+          <ul>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+          </ul>
```

Comment:
I think it could have been better if you put this `ul` in a `nav` container and place it in the header section.